app/config/security.yml

```diff
@@ -4,7 +4,6 @@ security:
     # http://symfony.com/doc/current/book/security.html#encoding-the-user-s-password
     encoders:
         Symfony\Component\Security\Core\User\User: plaintext
-        DashboardHub\Bundle\AppBundle\Entity\User: plaintext
```

Comment:
We can't remove it, it is required by the OAuth bundle, but I guess it doesn't do the encoding because it doesn't store the information itself & leaves it to the OAuth Service (in this case Github)